Src/stm32l5xx_hal_rcc.c

```diff
@@ -1375,6 +1375,9 @@ HAL_StatusTypeDef HAL_RCC_ClockConfig(RCC_ClkInitTypeDef  *RCC_ClkInitStruct, ui
 void HAL_RCC_MCOConfig(uint32_t RCC_MCOx, uint32_t RCC_MCOSource, uint32_t RCC_MCODiv)
 {
   GPIO_InitTypeDef GPIO_InitStruct;
+   
+  UNUSED(RCC_MCOx);
```

Comment:
Seems relevant particularly if macro `USE_FULL_ASSERT` is not defined (by default) or defined to be `0` as shown below

https://github.com/STMicroelectronics/stm32l5xx_hal_driver/blob/5e2bad9f15e03190274058bfb6df6f5f848eed0e/Inc/stm32l5xx_hal_conf_template.h#L179

This would yield the below definition:

https://github.com/STMicroelectronics/stm32l5xx_hal_driver/blob/5e2bad9f15e03190274058bfb6df6f5f848eed0e/Inc/stm32l5xx_hal_conf_template.h#L429